plugins/ranching_farm.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ranching_farm
```

Comment:
sorry for the late review. have you thought of picking a name like `ranching`, `ranching-farm` which are a bit more straightforward?

Reply:
i can do that, i'm not 100% married to it yet :)